packages/backend/src/peripherals/database/migrations/065_total_supply.ts

```diff
@@ -0,0 +1,40 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('total_supplies', function (table) {
+    table.dateTime('unix_timestamp').notNullable()
+    table.decimal('total_supply', 80, 0).notNullable()
+    table.string('asset_id').notNullable()
+    table.integer('chain_id').notNullable()
+
+    table.primary(['unix_timestamp', 'asset_id', 'chain_id'])
```

Comment:
change the order to [chain_id, unix_timestamp, asset_id]

Reply:
https://github.com/l2beat/l2beat/pull/1663/commits/80edc53aee622756e426907bbe0ef571f70aea99